### ISSUE TITLE
MATCH error in Query

### ISSUE THREAD

**Issue by edgesoft** [neo4j-graphql/neo4j-graphql-js#219]
Hi! I'm using the following dependencies

```js
"apollo-server": "^2.4.8",
"graphql": "^14.1.1",
"graphql-auth-directives": "^2.1.0",
"neo4j-driver": "^1.7.3",
"neo4j-graphql-js": "^2.4.2"
```

I have this type

```js
type TG_DAY {
  id: Float!
  date: String
  day: Int
  month: Int
  year: Int
}
```

When I run the Cyper query in neo4J browser everthing is fine

![Skärmavbild 2019-03-21 kl  11 48 15](https://user-images.githubusercontent.com/6061720/54747474-56e9ff00-4bcf-11e9-9a1a-12fb3941d631.png)

But this query in Grapiql results in error
```js
{
  TG_DAY (first: 1) {
    date
    day
  }
}
```

The debug says this in console
MATCH (`tG_DAY`:`TG_DAY` ) RETURN `tG_DAY` { .date , day: `undefined`.undefined.day } AS `tG_DAY` SKIP $offset LIMIT $first
{ offset: 0, first: 1 }

There seems to be something wrong with the mapping

**Comment by edgesoft** [comment]
day: undefined.undefined.day

**Comment by johnymontana** [comment]
Looks like we might be improperly identifying `day` as a temporal type element (instead of treating it as an int). 

Note that we do support the native Neo4j temporal types: https://grandstack.io/docs/neo4j-graphql-js.html#temporal-types-datetime

**Comment by johnymontana** [comment]
Fixed in #228, please reopen if you are still seeing this behavior.
